bin/varnishd/cache/cache_backend.c

```diff
@@ -690,7 +697,8 @@ static const struct vdi_methods vbe_methods[1] = {{
 	.destroy =		vbe_destroy,
 	.panic =		vbe_panic,
 	.list =			vbe_list,
-	.healthy =		vbe_healthy
+	.healthy =		vbe_healthy,
+	.notify =		vbe_notify
```

Comment:
I was bothered by the notify method because this is a breaking change as we established earlier and I wasn't satisfied with such a resolution on the 7.6 branch where we'd rather not break VRT if we can help it.

It then occurred to me that in spirit the new notify method is very close to the event method and I was wondering whether we'd agree to the creation of a new going-sick event even though that was not the original scope of the event enum.

I don't personally see a strong reason against widening the scope beyond the existing set of events.

Reply:
Is that yes, no or something else, @dridi ?

Reply:
Ah, phk also brought up using the `.event` method, is this what you mean? I can look at it, but it did not seem natural to me.

Reply:
I was suggesting adding a new enum entry for the event callback, one that would be dedicated to backend/directors, instead of adding a new notify callback.

Reply:
FTR: I did consider using the .`event()` method, but it seemed confusing to me because it is tied to the VCL temperature / lifetime. But as both you and phk do not share this concern, I will implement this to have something to look at.

Reply:
Thanks!